.composer-require-checker.json

```diff
@@ -12,6 +12,7 @@
         "PHPUnitGoodPractices\\Traits\\IdentityOverEqualityTrait",
         "PHPUnitGoodPractices\\Traits\\ProphecyOverMockObjectTrait",
         "PHPUnitGoodPractices\\Traits\\ProphesizeOnlyInterfaceTrait",
+        "LegacyPHPUnit\\TestCase",
```

Comment:
Can we have this higher, so the list is still alphabetically sorted?

Reply:
Good point. I'll make it happen.

Reply:
Should be all proper now.